app/src/main/AndroidManifest.xml

```diff
@@ -12,19 +12,26 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Evently">
         <activity
-            android:name=".ui.auth.AuthActivity"
+            android:name=".ui.entrant.EntrantActivity"
```

Comment:
Gotta switch this back and make `AuthActivity` transition into `EntrantActivity`